### PR TITLE
new upgrade scenarios added

### PR DESCRIPTION
**Pull request by abvgedeika**

### CODE REVIEW

File: data_dir/scylla.repo.upgrade_2.0 [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[scylla]
+name=Scylla for Centos $releasever - $basearch
+baseurl=http://downloads.scylladb.com/rpm/unstable/centos/branch-2.0/8/scylla/$releasever/$basearch/
```
Comment (amoskong) [inline data_dir/scylla.repo.upgrade_2.0]: Can we use the `latest one` (http://downloads.scylladb.com/rpm/unstable/centos/branch-2.0/latest/)? then it's always the latest.

**Comment by lmr** [comment]
Simple comment here, I spotted a typo, should be 'scenarios' instead of 'scenarious'.

**Comment by larisau** [comment]
why you do not use commit --amend to push minor(without specific meaning) commits?

**Comment by larisau** [comment]
What's the thrift_bindings module and why do we need to copy it to our project?

**Comment by abvgedeika** [comment]
@larisau thrift_bindings modules are used in schema_upgrade_tests.py when we use the thrift client in upgrade tests. these modules where copied from dtest

**Comment by larisau** [comment]
so maybe it's better to clone dtest in jenkins job, and configure the test to use it instead of copying the thrift module to this project

**Comment by abvgedeika** [comment]
@larisau if clone only in jenkins we will get no_module issue in all other cases + it's an additional manual step where these modules required

**Pull request merged**